app/api/routers/breeze_buddy/users/handlers.py

```
@@ -0,0 +1,487 @@
+
+
+async def update_user_handler(
+    current_user: UserInfo,
+    user_id: str,
+    user_data: UserUpdateRequest,
+) -> UserResponse:
+    """PUT /user/{id} - Update user by UUID."""
+    require_admin_or_merchant(current_user)
+    
+    # Check ownership
+    target_user = await check_user_ownership(current_user, user_id)
```

Ruff (0.14.13)

Local variable `target_user` is assigned to but never used

Remove assignment to unused variable `target_user`

(F841)